src/LauthzServiceProvider.php

```diff
@@ -56,5 +58,25 @@ public function register()
         $this->app->bind(ModelLoader::class, function($app, $config) {
             return ModelLoaderFactory::createFromConfig($config);
         });
+
+        $this->registerGates();
+    }
+
+    /**
+     * Register a gate that allows users to use Laravel's built-in Gate to call Enforcer.
+     *
+     * @return void
+     */
+    protected function registerGates()
+    {
+        Gate::define('enforcer', function ($user, ...$args) {
+            $identifier = $user->getAuthIdentifier();
+            if (method_exists($user, 'getAuthzIdentifier')) {
+                $identifier = $user->getAuthzIdentifier();
+            }
+            $identifier = strval($identifier);
+
+            return Enforcer::enforce($identifier, ...$args);
+        });

```

Comment:
This does not make full use of the Gate. Here the policy should be registered to the `Gate`.
We can check if a user has a permission with Laravel's default can function: `$user->can('edit');`

Therefore, the Gate check should be intercepted here:
```php
Gate::before(function (User $user, string $act, $obj...) {});

Gate::after(function (User $user, string $act, $obj...) {});
```
See: https://laravel.com/docs/11.x/authorization#intercepting-gate-checks

And then
1. Whether to use `Gate` is a configurable item
2. Use `after` or `before` interception is also a configurable item